contract_comm/blockchain_parameters/blockchain_parameters.go

```diff
@@ -67,21 +67,36 @@ const (
 		"payable": false,
 		"stateMutability": "view",
 		"type": "function"
-	  },
-	  {
+	},
+	{
 		"constant": true,
 		"inputs": [],
-		"name": "intrinsicGasForAlternativeFeeCurrency",
+		"name": "getUptimeLookbackWindow",
 		"outputs": [
-		  {
-			"name": "",
-			"type": "uint256"
-		  }
+			{
+				"internalType": "uint256",
```

Comment:
I don't know much about ABI.   Is this internalType necessary?  Does it make a difference?  Should we have it elsewhere as well?  Is it generated by another tool in which internalType was added in a newer version?

Reply:
we normally copy and paste the output of truffle compile. For that i wouldn't change it